src/modules/users/users.service.ts

```
@@ -1521,7 +1608,7 @@ export class UserService {
       let updatedDocData;
 
       try {
-        updatedDocData = await this.fetchVcJsonFromUrl(userDoc.doc_data_link);
+        updatedDocData = await this.fetchVcJsonFromVcUrl(userDoc.doc_data_link);
```

ESLint

[error] Replace `········updatedDocData·=·await·this.fetchVcJsonFromVcUrl(userDoc.doc_data_link` with `↹↹↹↹updatedDocData·=·await·this.fetchVcJsonFromVcUrl(⏎↹↹↹↹↹userDoc.doc_data_link⏎↹↹↹↹`

(prettier/prettier)